zerver/lib/alert_words.py

```diff
@@ -46,25 +56,30 @@ def get_alert_word_automaton(realm: Realm) -> ahocorasick.Automaton:
     return alert_word_automaton
 
 
-def user_alert_words(user_profile: UserProfile) -> List[str]:
-    return list(AlertWord.objects.filter(user_profile=user_profile).values_list("word", flat=True))
+def user_alert_words(user_profile: UserProfile) -> List[Dict[str, str]]:
```

Comment:
Should be `List[AlertWordData]`. Pydantic models can be serialized as well as deserialized—there should be no need to manually construct dicts for them.